packages/client/src/app/shared/title.ts

```diff
@@ -0,0 +1,17 @@
+import { from } from 'rxjs';
+import { Constants } from './constants';
+
+export namespace Title {
+    export function setDefaultTitle(): void {
+        document.title = Constants.OFFPRINT;
+    }
+
+    export function setTwoPartTitle(title: string): void {
+        document.title = title + Constants.SEPARATOR + Constants.OFFPRINT;
+    }
+
+    export function setThreePartTitle(title1: string, title2: string): void {
+        document.title = title1 + Constants.SEPARATOR + title2 +
```

Comment:
suggestion/idea/thought: Javascript/TypeScript support string interpolation, so you could also write this as
```typescript
document.title = `${title1} ${Constants.SEPARATOR}$ ${title2} ${Constants.SEPARATOR} ${Constants.OFFPRINT}`;
```
...which is a little hard to read, and (personal opinion) I think you're better off not defining a separator constant anyway. Then you could do:
```typescript
document.title = `${title1} - ${title2} - ${Constants.OFFPRINT}` ;
```